app/controllers/application_controller.rb

```diff
@@ -2,4 +2,5 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  layout 'site'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.